### ISSUE TITLE
a11y manager integration with flutter

### ISSUE THREAD

**Issue by kf6gpe** [flutter/flutter#35831]
This involves wiring up flutter to the a11y framework via the flutter bridge. The accessibility manager should be notified when a view is added and given an appropriate id (shared with scenic) to identify it. 

This is a mirror issue for https://fuchsia.atlassian.net/browse/MI4-1539; conversation should please occur there.

**Comment by Hixie** [comment]
https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=18191

**Comment by Hixie** [comment]
FL-250 (testing thereof in particular) still taking priority as of today.

**Comment by dnfield** [comment]
This is fixed. We will have additional work to do for a11y on Fuchsia, but that will be tracked in different issue(s).
